tests/tests/tier0/bluechi-agent-connect-via-manager-address/test_bluechi_agent_connect_via_manager_address.py

```diff
@@ -0,0 +1,66 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+from typing import Dict
+import os
+import socket
+
+
+from bluechi_test.test import BluechiTest
+from bluechi_test.machine import BluechiControllerMachine, BluechiAgentMachine
+from bluechi_test.config import BluechiControllerConfig, BluechiAgentConfig
+from bluechi_test.util import read_file
+
+NODE_FOO = "node-FOO"
+failed_status = "failed"
+
+
+def exec(ctrl: BluechiControllerMachine, nodes: Dict[str, BluechiAgentMachine]):
```

Comment:
This method should contain only following:

1. Verify that agent on node foo is started and connected to controller
2. Copy config file with incorrect host in `ControllerAddress` to node foo
3. Restart agent with config file loaded from command line parameter
4. Verify that agent is started, but it cannot connect to the controller